Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -340,6 +342,16 @@ public function testDeprecatedRenderListElement($expected, $value, array $option
         );
     }
 
+    public function buildUrl($url)
+    {
+        if (defined('PHP_QUERY_RFC3986')) {
+            // add htmlspecialchars because twig add it auto
```

Comment:
Oh ok I get it now! For some reason I thought this was in a new Twig filter you wrote… I must be tired!